java/de/yessoft/android/mohammad/application/app.java

```diff
@@ -0,0 +1,58 @@
+package de.yessoft.android.mohammad.application;
+
+import android.app.Application;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.os.Build;
+
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.FirebaseFirestoreSettings;
+
+public class app extends Application {
+
+    public static final String CHANNEL_1_ID = "channel1";
+    public static final String CHANNEL_2_ID = "channel2";
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        // region Enable Offline Data
+        FirebaseFirestoreSettings settings = new FirebaseFirestoreSettings
```

Comment:
firebase code should moved out, to separate class ,
and here in the constructor we should accept only interface , 
 firebase  implementation of the interface then can be injected to this class

Reply:
OK